stdlib/public/core/HashedCollections.swift.gyb

```diff
@@ -2513,6 +2805,47 @@ internal struct _Native${Self}Storage<${TypeParameters}> {
   /// We store a RawBuffer so that this can be an empty singleton
   internal var _buffer: RawBuffer
 
+  /// Creates a Storage with a buffer that is typed, but doesn't understand
+  /// Hashing. Mostly for bridging; prefer `create(capacity:)`.
```

Comment:
Many operations on the hash table don't actually involve hashing at all. Indexing and iterating being notable examples. You just walk over the structure. In our particular case, we create a `NativeStorage<AnyHashable, AnyHashable>`, and use the indices from `NativeStorage<K, V>` to do lookups in it when doing bridging to objc.